### ISSUE TITLE
get wrong css order after using HashedModuleIdsPlugin

### ISSUE THREAD

**Issue by silaike** [webpack/webpack#5733]
**Do you want to request a *feature* or report a *bug*?**
Bug.

**What is the current behavior?**
get wrong css order after using HashedModuleIdsPlugin

**If the current behavior is a bug, please provide the steps to reproduce.**
webpack plugin configs.
```javascript
       ...
	plugins: [
		new HtmlWebpackPlugin( {
			filename: 'index.html',
			template: 'index.ejs',
			minify: {
				collapseWhitespace: true,
				minifyJS: true
			}
		} ),

		new ExtractTextPlugin( {
			filename: 'css/[name].[contenthash:8].css',
			allChunks: true,
		} ),

		new OptimizeCssAssetsPlugin( {
			cssProcessor: require( 'cssnano' ),
			cssProcessorOptions: {
				autoprefixer: false
			}
		} ),

		new webpack.HashedModuleIdsPlugin(),

		new webpack.optimize.CommonsChunkPlugin( {
			names: [ 'vendor', 'manifest' ],
		} ),

		new webpack.optimize.CommonsChunkPlugin( {
			async: 'async-common',
			minChunks: ( module, count ) => (
				count >= 2
			),
		} ),

		new webpack.DefinePlugin( {
			'process.env': {
				NODE_ENV: JSON.stringify( 'production' )
			}
		} ),

		new webpack.optimize.UglifyJsPlugin( {
			compress: {
				warnings: false
			},
			comments: false
		} ),

		new webpack.optimize.ModuleConcatenationPlugin(),
	]
```

**What is the expected behavior?**
remove HashedModuleIdsPlugin then css order will be right.

**Please mention other relevant information such as the browser version, Node.js version, webpack version and Operating System.**
webpack version v3.6.0

**Comment by Kubaka1** [comment]
Same issue +1... @sokra can help this?

**Comment by cletusw** [comment]
Likely fix merged: d5a1de21a65532f66b05a1f6c2146574ce385567

**Comment by evilebottnawi** [comment]
@webpack-bot move to `webpack-contrib/extract-text-webpack-plugin`

Failed test https://github.com/webpack-contrib/extract-text-webpack-plugin/pull/710
